.gitignore

```diff
@@ -11,3 +11,4 @@ test/services/tokens
 # IDE
 /*.iml
 .idea
+.vscode
```

Comment:
[Code style] We tend to have a newline at the end of our files.

You can configure your editor to do so on file save.
I am pretty sure you already know that, sorry if this is an obvious comment. ;)

Maybe we should lint that to prevent that kind of review comment in our repositories.

Reply:
Done, thanks.

I changed the configuration in my IDE